crates/fee/src/lib.rs

```diff
@@ -268,32 +260,17 @@ pub mod pallet {
     // The pallet's dispatchable functions.
     #[pallet::call]
     impl<T: Config> Pallet<T> {
-        /// Withdraw vault rewards.
-        ///
-        /// # Arguments
-        ///
-        /// * `origin` - signing account
-        #[pallet::weight(<T as Config>::WeightInfo::withdraw_vault_rewards())]
-        #[transactional]
-        pub fn withdraw_vault_rewards(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
-            ext::security::ensure_parachain_status_not_shutdown::<T>()?;
-            let signer = ensure_signed(origin)?;
-            Self::withdraw_from_pool::<T::VaultRewards>(RewardPool::Global, &signer)?;
-            Ok(().into())
-        }
-
-        /// Withdraw nominator rewards.
+        /// Withdraw rewards.
```

Comment:
Is this function only for nominators? Should update the doc comment in this case.